update SCIM docs for SAML2 Providers (#4505)
getsentry/sentry-docs: src/docs/product/accounts/sso/saml2.mdx

Because Sentry uses Just-In-Time (JIT) provisioning, new members are registered with Sentry automatically during their first login attempt with SAML SSO. These accounts will have their membership types delegated based on what the default membership role is in your Organization Settings in Sentry.

> ## SCIM Provisioning
> SCIM is available for automatic user provisioning, deprovisioning, and team assignment for SAML2 IdPs. See the Sentry configuration guide [here](https://docs.sentry.io/product/accounts/sso/okta-sso/okta-scim/#sentry-configuration), and follow your IdP's documentation for further setup. For more information about the Sentry SCIM API endpoints, see [here](https://docs.sentry.io/api/scim/).
> 
> 

Reviewer: ```suggestion
## SCIM Provisioning

SCIM is available for automatic user provisioning, deprovisioning, and team assignment for SAML2 IdPs. [Configure Sentry for SCIM](https://docs.sentry.io/product/accounts/sso/okta-sso/okta-scim/#sentry-configuration), and then follow your IdP's documentation for further setup. Learn more about the Sentry SCIM API endpoints in the [SCIM API documentation](https://docs.sentry.io/api/scim/).

```
